### DISCUSSION TITLE
Share buttons

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#1442] [Q&A] [answered]
Hi, 

Is possible to include share buttons in each blog post so readers could share an article that I am writing in social media? (Linkedin, Twitter, Facebook etc.) 

Thanks

**Comment by dragonstyle** [comment, marked as answer]
Right now we don't have a built in feature for generating sharing buttons on posts, though this is a great idea. I'll convert this into an issue to track that feature request.

In the meantime, you could consider writing a Quarto extension to support this or just add appropriate links to your blog page, footer, or elsewhere.
